Sources/FigmaExport/Loaders/ImagesLoader.swift

```diff
@@ -165,6 +165,16 @@ final class ImagesLoader {
         return imagePacks
     }
 
+    private func getScales(platform: Platform) -> [Double] {
+        if platform == .android {
+            return [1, 2, 3, 1.5, 4.0]
+        } else {
+            let validScales: [Double] = [1, 2, 3]
+            let customScales = params.ios?.images.scales?.filter { validScales.contains($0) } ?? []
+            return !customScales.isEmpty ? customScales : validScales
```

Comment:
Do not use «!» here because it reduces readability when using `? :`.
`return customScales.isEmpty ? validScales : customScales`

Reply:
Done